### ISSUE TITLE
​Files within the workspace are shared individually instead of sharing the entire workspace

### ISSUE THREAD

**Issue by Zijinn** [toeverything/AFFiNE#8893]
### Description

It would be great if the software supports separate sharing of files in the workspace or supports the preservation of personal privacy in the workspace, so that you don't have to switch between collaborative files and personal files in the workspace.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by forehalo** [comment]
on the table

**Comment by JimmFly** [comment]
![Image](https://github.com/user-attachments/assets/fc61ee91-2d38-47ed-a47f-7e8c094dd745)

In the latest version v0.21, you can control the permissions of the current document in the sharing menu. Does this meet your requirements?

**Comment by Zijinn** [comment]
Yes, this is the function I want. I think it will greatly provide flexible collaboration for sharing files, but I found that it is only available in the pro version. I wanted to support our lifetime plan before, but it is still a bit expensive for a student. Thank you for the optimization and improvement you have made to the software, thank you!
